Add imu manager package documentation (#126)
project-march/tutorials: doc/march_packages/march_imu_manager.rst

.. _march-imu-manager-label:

march_imu_manager
=================

Overview
--------
The ``march_imu_manager`` interfaces between ROS and the `XSense MTw Awinda IMU's <https://www.xsens.com/products/mtw-awinda>`_.

> The IMU's connect wireless to an USB dongle, which should be inserted in the computer the node is running on.

Reviewer: ```suggestion
The IMU's connect wireless to an USB dongle, which should be inserted in the computer before the node is launched.
```
